src/ResourceManager/Insights/Commands.Insights/Microsoft.Azure.Commands.Insights.Types.ps1xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
In fact, that file I was using to experiment with the display of the results.
Should I remove it even if it is empty?